service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java

```diff
@@ -121,13 +122,29 @@ public UsersRecord createNewUser(
     newUser.setPrivilegeLevel(PrivilegeLevel.STANDARD);
     newUser.store();
 
+    AuditRecord audit = db.newRecord(AUDIT);
+    audit.setTableName("users");
+    audit.setTransactionType("insert");
```

Comment:
We should try to use Enums for these values. jOOQ may or may not provide something that is usable here (table names).